examples/apptainer.yaml

```diff
@@ -24,10 +30,10 @@ provision:
     #!/bin/bash
     set -eux -o pipefail
     command -v apptainer >/dev/null 2>&1 && exit 0
-    dnf -y install apptainer
-    # See https://fedoraproject.org/wiki/Features/tmp-on-tmpfs
-    echo "APPTAINER_TMPDIR=/var/tmp" > /etc/profile.d/apptainer.sh
```

Comment:
note: the ubuntu installation has a normal /tmp (not tmpfs), so doesn't need this override